### ISSUE TITLE
Add a scale to the optical axis

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/geometric-optics#357]
3/3/2022 for #328

@kathy-phet suggested that adding an optional scale to the optical axis might address some of the measurement issues that @ariel-phet raised in #334.  

Notes:
* add an additional checkbox, or show when labels are shown, or use the "Values" checkbox proposed in #334, or something else?
* tick marks and values might be the same as the major tick marks on the ruler
* tick marks and values would need to update to match zoom level and browser window width

We decided not to do this for release #278, deferred for consideration in a future release.

**Comment by kathy-phet** [comment]
Again continue to defer this for now. It does seem like it is still a viable issue for the measurement tool challenge. But let's stay focused on releasing what we have as PhET-iO, and wait for user driven requests.

**Comment by pixelzoom** [comment]
Agreed. We should not address this for #278, the 1.3 PhET-iO release.
